packages/cedar-amcharts/src/render/render.ts

```diff
@@ -98,6 +98,13 @@ export function fillInSpec(spec: any, definition: any) {
 }
 
 export function fetchSpec(type: string): any {
+  if (type === 'time') {
```

Comment:
Suggest using `let spec = type` and then overriding if needed in the `if` blocks w/ `spec = 'line'`, etc. b/c we're going to have at least more of these ('grouped').